libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -159,8 +159,6 @@
 		CE4CE35E1C0E526A009F6029 /* SFAuthErrorHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = 4F96FCF31BFD32130022F021 /* SFAuthErrorHandler.m */; };
 		CE4CE35F1C0E526A009F6029 /* SFAuthErrorHandlerList.h in Headers */ = {isa = PBXBuildFile; fileRef = 4F96FCF41BFD32130022F021 /* SFAuthErrorHandlerList.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		CE4CE3601C0E526A009F6029 /* SFAuthErrorHandlerList.m in Sources */ = {isa = PBXBuildFile; fileRef = 4F96FCF51BFD32130022F021 /* SFAuthErrorHandlerList.m */; };
-		CE4CE3611C0E526A009F6029 /* SFAuthorizingViewController.h in Headers */ = {isa = PBXBuildFile; fileRef = 4F96FCF61BFD32130022F021 /* SFAuthorizingViewController.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		CE4CE3621C0E526A009F6029 /* SFAuthorizingViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 4F96FCF71BFD32130022F021 /* SFAuthorizingViewController.m */; };
```

Comment:
Cool.  The only other place that I know of that it needs to be removed is in SalesforceSDKCore.podspec.